.github/actions/e2e-headless-ssr/action.yml

```diff
@@ -5,15 +5,21 @@ inputs:
     description: The GitHub token for accurately detecting a build vs a re-run build
     required: false
     default: ''
+  workspace:
+    description: The NPM workspace to run the tests in
+    required: true
+  mode:
+    description: The mode to run the tests in
+    required: true
 runs:
   using: composite
   steps:
     - uses: cypress-io/github-action@v5
       name: Run Cypress
       with:
         browser: chrome
-        command-prefix: 'npm run -w @coveo/headless-ssr-samples-app-router e2e'
-        start: npx nx run headless-ssr-samples-app-router:dev
+        command-prefix: 'npx -w @coveo/${{ inputs.workspace }} cypress-repeat run -n 3 --until-passes --rerun-failed-only --'
```

Comment:
Why use cypress-repeat instead of built-in cypress test retries? https://docs.cypress.io/guides/guides/test-retries#Configure-Test-Retries

As I understand it, cypress-repeat is mainly meant to help detecting flaky tests.

Reply:
We use both irc, but cypress-repeat removes tests that worked from the retries, which the option does not